### DISCUSSION TITLE
Application approved but no Pro

### DISCUSSION THREAD

**Discussion by taz717** [community/community#77969] [GitHub Education]
### Hi everyone,

Hello, I my application for the GitHub Education got approved last month; however, while waiting for it to get approved, I got a month of pro on my own. Now that the month is over, I don't have access to Pro and I think that might be related to why I no longer have it anymore. Is there a way to renew it using GitHub Education or should it have been automatically applied?

![image](https://github.com/community/community/assets/48331207/0fcc8bcf-b363-4dd7-a6e4-dddb721d67ed)

![image](https://github.com/community/community/assets/48331207/4d38e959-e7c4-44ad-bb29-1f48ce543121)

**Comment by resslr** [comment]
Contact support, they might be able to help you.

I can try to help you, I have some questions:
How long ago did it say you got access to your benefits?

**Reply by gauthamdas** [reply]
@taz717 Did you get Pro?. Even I'm facing the same issue.
